### ISSUE TITLE
Express graduation to impact project

### ISSUE THREAD

**Issue by UlisesGascon** [openjs-foundation/cross-project-council#1378]
Ref: https://github.com/openjs-foundation/project-status/issues/15#issuecomment-2410399877

As discussed today in the CPC meeting, the idea is to work on this in the following weeks and evaluate the process to follow and requirements.

**Comment by ctcpip** [comment]
today in the CPC working session, we went through the Impact stage acceptance criteria and are ready to present for consideration by the CPC at the next meeting:

- Document that it is being used successfully in production by at least two independent end users which, in the CPC’s judgement, are of adequate quality and scope.
  - Express is widely deployed in production and CPC working session folks agree this is adequate justification for this item.
  - 34,419,879 weekly downloads on npm as of 2024-10-22
  - 1,353,331 IPs using the HTTP header `x-powered-by: Express` based on [Shodan engine](https://www.shodan.io/search?query=x-powered-by%3A+Express) this does not include applications using middleware like Helmet or custom http header rules.
  - [Independent analysis](https://enlyft.com/tech/products/express-js) 
  - The full Express / pillarjs / jshttp ecosystem is around 52B downloads per year ([data from 2023-2024](https://docs.google.com/spreadsheets/d/1Qi7B78K6R_RyFloAcrcizL2oYawwMEmdKjgzC_Lik9Q/edit?usp=sharing)).
- Have a defined governing body of at least 5 or more members (owners and core maintainers), of which no more than 1/3 is affiliated with the same employer. In the case there are 5 governing members, 2 may be from the same employer.
  - TC:
    - Blake Embrey
    - Chris de Almeida (IBM)
    - Jean Burellier (Sanofi)
    - Jon Church (Microsoft)
    - Linus Unnebäck
    - Rand McKinney (Adobe)
    - Ulises Gascón (NodeSource)
    - Wes Todd (Netflix)
- Have a documented and publicly accessible description of the project's governance, decision-making, contribution, and release processes. This is preferably laid out in a GOVERNANCE.md file and references a CONTRIBUTING.md and AUTHORS.md file showing the current and emeritus committers.
  - https://github.com/expressjs/express/blob/master/Charter.md
  - https://github.com/expressjs/express/blob/master/Collaborator-Guide.md
  - https://github.com/expressjs/express/blob/master/Contributing.md
  - https://github.com/expressjs/express/graphs/contributors
- Demonstrate a substantial ongoing flow of commits and merged contributions.
  - https://github.com/expressjs/express/commits/master/
  - https://github.com/expressjs/express/pulse/monthly
  - LFX Insights: [Express](https://insights.lfx.linuxfoundation.org/foundation/ojsf/overview/github?project=ojsf-express&routedFrom=Github&bestPractice=false), [jshttp](https://insights.lfx.linuxfoundation.org/foundation/ojsf/overview/github?project=jshttp&routedFrom=Github&bestPractice=false&repository=) and [pillarjs](https://insights.lfx.linuxfoundation.org/foundation/ojsf/overview/github?project=pillarjs&routedFrom=Github&bestPractice=false&repository=)
- Have a healthy number of committers from at least two organizations. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
  - Github Contributors: https://github.com/expressjs/express/graphs/contributors?from=10%2F21%2F2023
- Adopt the Foundation Code of Conduct.
  - ✅ https://github.com/expressjs/express/blob/master/Code-Of-Conduct.md
- Accept security reports and publicly disclose them after the fix has been made available.
    - ✅ https://github.com/expressjs/express/security/advisories?state=published
    - ✅ https://expressjs.com/2024/09/29/security-releases.html
- Have a public list of project adopters for at least the primary repo (e.g., ADOPTERS.md or logos on the project website).
    - as discussed in CPC working session, this seems out of place, and we'll create a PR to remove this criterion
- Other metrics as defined by the applying Project during the application process in cooperation with the CPC.
  - https://github.com/expressjs/discussions/issues/160
- Receive a supermajority vote from the CPC to move to Impact stage. Projects can move directly from At Large to Impact, if they can demonstrate sufficient maturity and have met all requirements.
  - 🙏

**Comment by tobie** [comment]
This is really sweet. Well done, folks!

**Comment by anfibiacreativa** [comment]
wow, congrats!

**Comment by tobie** [comment]
**Notes from this week's CPC call:**

* Process for approval to move to Impact still requires voting
* We'll vote and and change the process for next time (so it is consensus-driven)

Action items:

- [x] email needs to be sent (@mhdawson)
- [x] kickoff voting (@bensternthal)
- [x] Express to open PR against README (@ctcpip)
- [x] update website (Ben)
- [x] Open issue to align process with current non-voting practices (@tobie) (See: https://github.com/openjs-foundation/cross-project-council/issues/1388)
- [x] PR to update acceptance criterion re: decision-making (@ctcpip)
- [x] add express voting members to website (@bensternthal)

**Comment by ctcpip** [comment]
@tobie @bensternthal did the vote occur?

**Comment by bensternthal** [comment]
@ctcpip Yes this vote is complete and this passed by over 2/3. ![Image](https://github.com/user-attachments/assets/cddf899c-950f-472f-b35d-8e8f951e6e38)

**Comment by ctcpip** [comment]
Fantastic! I will commence with the action items above. Thanks everyone!

**Comment by jonchurch** [comment]
Looks like we still need to update the CPC readme project list, and ensure the OpenJSF Website has been updated to list Express as impact. cc @ctcpip

**Comment by ctcpip** [comment]
no, we still have [a couple items](https://github.com/openjs-foundation/cross-project-council/issues/1378#issuecomment-2452669162) to take care of

**Comment by bjohansebas** [comment]
any updates?

**Comment by UlisesGascon** [comment]
Is there anything I can do to help move this forward? Looks like only two items are left: https://github.com/openjs-foundation/cross-project-council/issues/1378#issuecomment-2584268441
